package.json

```diff
@@ -38,7 +38,7 @@
     "uglify-js": "^3.3.10"
   },
   "scripts": {
-    "test": "jasmine --config=jasmine.json",
+    "test": "node test && jasmine --config=jasmine.json",
```

Comment:
`test/specs/specs-spec.js` already takes care of running the `test/index.js` tests in jasmine so there is no need to run `node test`.